test/calibration/experiments/test_drag.py

```diff
@@ -25,11 +27,43 @@
 
 from qiskit_experiments.exceptions import CalibrationError
 from qiskit_experiments.library import RoughDrag, RoughDragCal
-from qiskit_experiments.test.mock_iq_backend import DragBackend
+from qiskit_experiments.test.mock_iq_backend import MockIQBackend
 from qiskit_experiments.calibration_management.basis_gate_library import FixedFrequencyTransmon
 from qiskit_experiments.calibration_management import Calibrations
 
 
+def compute_probability(
```

Comment:
This method should move to `qiskit_experiments/test/mock_iq_backend.py`, because users may need it

Reply:
I guess that `DragBackend` will be removed, and that the code here is copied from there?

Reply:
Yes. `DragBackend` will be removed and this function is part of the test because in the new design, the probability calculation function is provided by the user.